rwengine/src/render/GameRenderer.cpp

```diff
@@ -212,6 +212,7 @@ GameRenderer::GameRenderer(Logger* log, GameData* _data)
 
 GameRenderer::~GameRenderer() {
     glDeleteFramebuffers(1, &framebufferName);
+    glDeleteProgram(ssRectProgram);
```

Comment:
Or maybe should we convert ssRectProgram to  ShaderProgram?

Reply:
It should be doable. But isn't required for this PR.